### PR TITLE
doc: generating stone prover

### PR DESCRIPTION
**Pull request by Kaminar-i**

### CODE REVIEW

**Comment by Kaminar-i** [comment]
hello @dmirgaleev please kindly review

File: docs/pages/usage/proving.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Download the Docker image. The stone-prover package includes both `cpu_air_prover` and `cpu_air_verifier`:
+
+```bash
+docker pull ghcr.io/dipdup-io/stone-packaging/stone-prover:latest
+```
+
+## Creating and Verifying a Test Proof Using Docker
+
+- First you have to make that Docker is installed on your machine. You can refer to [Docker’s official documentation](https://docs.docker.com/get-docker/) for installation instructions and more informations.
```
Comment (dmirgaleev) [inline docs/pages/usage/proving.md:53]: Please, rephrase that. You can make it shorter.

File: docs/pages/usage/proving.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+1. Clone the Repository:
+
+```bash
+git clone https://github.com/dipdup-io/stone-packaging.git /tmp/stone-packaging
+```
+
+2. Run the Docker Container to generate the Proof
+
+Run the container with an amount of volume mounted to the local repository directory:
```
Comment (dmirgaleev) [inline docs/pages/usage/proving.md:63]: Let's rephrase that.

File: docs/pages/usage/proving.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+3. Run the verifier to validate the Proof Using Docker
+
+```bash
+docker run --entrypoint /bin/bash -v /tmp/stone-packaging/test_files:/app/prover ghcr.io/dipdup-io/stone-packaging/stone-prover -c "cd /app/prover && exec cpu_air_verifier --in_file=fibonacci_proof.json && echo 'Successfully verified example proof.'"
+```
+
+### Regular issues commonly found
+- **Docker not found:** Ensure Docker is installed correctly by running `docker --version`. If not installed, follow the [Docker installation guide](https://docs.docker.com/get-started/get-docker/).
+
+- **Docker Build Errors:** Make sure your Dockerfile syntax is correct and that you have a stable internet connection to download dependencies.
```
Comment (dmirgaleev) [inline docs/pages/usage/proving.md:85]: I think that's impossible. Please, chech what chatGPT gives you

**Review by dmirgaleev — CHANGES_REQUESTED** [review]

**Comment by Kaminar-i** [comment]
@dmirgaleev i have implemented requested changes 
and the docker build errors is possible

**Review by dmirgaleev — APPROVED** [review]

**Pull request merged**